sql/src/main/java/io/crate/execution/jobs/JobSetup.java

```diff
@@ -883,7 +883,7 @@ public Boolean visitNestedLoopPhase(NestedLoopPhase phase, Context context) {
                 joinCondition,
                 phase.joinType(),
                 breaker(),
-                phase.blockNestedLoop ? ramAccounting : ramAccountingOfOperation,
```

Comment:
Not sure why there was a special exception for blockNestedLoop. To me it seems like there is also a single thread that uses the `ramAccounting` instance.

Reply:
May it was just a mistake by me? I agree that only 1 thread will use it concurrently.